addons/l10n_fr/models/res_partner.py

```diff
@@ -0,0 +1,12 @@
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_fr_is_french_partner = fields.Boolean(compute='_compute_l10n_fr_is_french_partner')
```

Comment:
Oh and you could argue the _partner is superfluous, as it's a method on partner
```suggestion
    l10n_fr_is_french = fields.Boolean(compute='_compute_l10n_fr_is_french')
```